### ISSUE TITLE
Difference to .NET TimeSpan

### ISSUE THREAD

**Issue by DenisBiondic** [TimeXt/TimeXt-CSharp#1]
What is the difference to the .NET TimeSpan? I find the fluent syntax nice, but TimeSpan already offers many of the functionalities in the examples here, right? (like TimeSpan.FromSeconds, FromDays, operators are also overloaded etc. https://docs.microsoft.com/en-us/dotnet/api/system.timespan.fromseconds?view=netframework-4.7.2)

Also, wouldn't introducing a new type like Interval create problems when using this problem with existing libraries? Again, convention across .NET is to use TimeSpan / DateTime, so an implicit cast would be nice.

As a last point, TimeSpan is a struct for performance reasons, altough for 99% of projects out there will not make a difference.

**Comment by GuepardoApps** [comment]
You're right. There's nearly no difference there. I did this as a PoC after I created the Kotlin and JS version of this. TimeSpan already supports most of this, yes. 
Maybe I will change this lib in v0.2 to return TimeSpan using the fluent syntax. Something like 2.Days() shall then return the same as TimeSpan.FromDays(2).

**Comment by DenisBiondic** [comment]
Either that, or as an idea to consider, an implicit cast from interval to timespan could be implemented, so that you could do something like object.Duration = 2.Hours() + 5.Minutes(), given that Duration is of type timespan.

BTW, take a look at FluentAssertions library for your unit tests ;)

**Comment by GuepardoApps** [comment]
I made v0.2 as pure extensions for TimeSpan and added additional TimeValue Weeks. Also added FluentAssertions to the tests. 
And switched to .NET Core. Because division of TimeSpan is only supported since core if I'm not wrong.
Thanks for the input 👍

**Comment by DenisBiondic** [comment]
Cool. Perhaps the docs can be extended to show examples with TimeSpan, right? I would be happy to contribute with a pull request...

**Comment by GuepardoApps** [comment]
Would be nice. Feel free to contribute 😉

**Comment by DenisBiondic** [comment]
here is the pull request: https://github.com/conplementAG/FluentTimeSpan/pull/9
